src/modules/card/layout.jade

```diff
@@ -17,7 +17,9 @@ mixin card(data, options)
   if options.float
     - classes.push('card--' + options.float);
 
-  div(class=classes)
+  - attributes.class = classes;
```

Comment:
what about the classes you pass in the attributes? will they be there?

```
+card({}, {})(class=".a-card")
````

and this:

```
+card({}, {})(class=[".a-card"])
```